share/scripts/init-message.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+import os
+
+SHARE_ROOT = os.path.dirname(
+    os.path.dirname(
```

Comment:
Hehehe love it. I would rename to ./setup.py and put in the root directory to follow python conventions

Reply:
oooh yeah, thats right. Has been done :)